portfolio/src/main/webapp/index.html

```diff
@@ -2,18 +2,65 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Katherine Yu</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <h1 style="text-align:center">Katherine Yu</h1>
+      <div style="text-align:center">
+        <a href="https://www.linkedin.com/in/katherine-y-yu/" target="_blank"><img src="https://static-exp1.licdn.com/sc/h/akt4ae504epesldzj74dzred8" alt="LinkedIn" style="width:30px;vertical-align:middle;margin:0px 10px"></a>
+        <a href="https://github.com/katherine-yu" target="_blank"><img src="https://github.githubassets.com/favicons/favicon.svg" alt="GitHub" style="width:30px;vertical-align:middle;margin:0px 10px"></a>
+      </div>
+
+      <h2>About</h2>
+      <p>I am a Computer Science major at Stanford University, and I grew up in Winchester, MA, which is not far from Boston. I love to play the violin and study math&mdash;currently, I am quite enjoying courses in real analysis.</p>
+      
+      <p>Click here for a random fact about me: &nbsp; &nbsp;<button style="height:50px;width:50px;background-color:black;color:white" onclick="addRandomGreeting()">?</button></p>
+      <div id="random-fact-container"></div>
+      <br>
+
+      <h2>Gallery</h2>
+      <div class="container">
+        <button style="margin:20px" onclick="previousImg()"><</button>
+        <img src="images/ico1.jpg" alt="origami isocahedron" width="250" data-idx="0" id="galleryimg">
+        <button style="margin:20px" onclick="nextImg()">></button>
```

Comment:
Just as FYI, another way you can add functionality to operate on a click is using [DOM event listeners](https://www.w3schools.com/js/js_htmldom_eventlistener.asp) which will essentially listen out for an event on a particular element that you want. This can be particularly useful if you need to keep track of some context like you are here. Rather than adding data-idx, you can keep track of the index as a Javascript variable and increment/decrement it.

Event listeners can also be useful for example, if you need to pass in some data to your Javascript function,. It can also listen for events such as mouseover, mouseout, double click, etc...

This may come in handy in the future, but what you have now is perfectly fine! If you want to try out it just to get some practice using it, feel free to do this in a follow up PR, but completely optional.